org.unofficialrevport.REVHubInterface.yml

```diff
@@ -0,0 +1,69 @@
+id: org.unofficialrevport.REVHubInterface
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: REVHubInterface.sh
+finish-args:
+  - --device=all
+  - --share=ipc
+  - --socket=x11
+modules:
+  - python3-requirements.json
+  - name: tkinter
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix=${FLATPAK_DEST} .
+    sources:
+      - type: git
+        url: https://github.com/iwalton3/tkinter-standalone
+        commit: 88aa05075d90d393a29a484bce676e237d311082
+    modules:
+      - name: tcl
+        buildsystem: autotools
+        subdir: unix
+        post-install:
+          - chmod 755 /app/lib/libtcl*.so
+        cleanup:
+          - /bin
+          - /lib/pkgconfig
+          - /man
+        sources:
+          - type: archive
+            url: https://prdownloads.sourceforge.net/tcl/tcl8.6.14-src.tar.gz
+            sha256: 5880225babf7954c58d4fb0f5cf6279104ce1cd6aa9b71e9a6322540e1c4de66
+            x-checker-data:
+              type: anitya
+              project-id: 4941
+              stable-only: true
+              url-template: https://prdownloads.sourceforge.net/tcl/tcl$version-src.tar.gz
+      - name: tk
+        buildsystem: autotools
+        subdir: unix
+        post-install:
+          - chmod 755 /app/lib/libtk*.so
+        cleanup:
+          - /bin
+          - /lib/pkgconfig
+          - /man
+        sources:
+          - type: archive
+            url: https://prdownloads.sourceforge.net/tcl/tk8.6.14-src.tar.gz
+            sha256: 8ffdb720f47a6ca6107eac2dd877e30b0ef7fac14f3a84ebbd0b3612cee41a94
+            x-checker-data:
+              type: anitya
+              project-id: 11426
+              stable-only: true
+              url-template: https://prdownloads.sourceforge.net/tcl/tk$version-src.tar.gz
+  - name: revhubinterface
+    buildsystem: simple
+    builddir: true
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} . --no-build-isolation
+      - install -Dm755 REVHubInterface.sh /app/bin/REVHubInterface.sh
+      - install -Dm644 -t /app/share/applications org.unofficialrevport.REVHubInterface.desktop
+      - install -Dm644 -t /app/share/icons/hicolor/scalable/apps org.unofficialrevport.REVHubInterface.svg
```

Comment:
neither file can be found. Also there is no mandatory appstream file.

Reply:
I included them in the pr, where should they be put?

Reply:
Should it be in the directory of the actual software?

Reply:
You are still missing the metainfo file. https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#path-and-filename

Reply:
This is fixed in the current revision.